pkg/spec/tf_job.go

```diff
@@ -75,7 +79,8 @@ const (
 type ContainerName string
 
 const (
-	TENSORFLOW ContainerName = "tensorflow"
+	TENSORFLOW     ContainerName = "tensorflow"
+	PsDefaultImage               = "wbuchwalter/mlkube-tensorflow-ps"
```

Comment:
Can we default to a TensorFlow image now? Can we make this a flag to make it easily customizable?

Reply:
Sorry this is something I forgot to remove from my first implementation (with custom images).
Now, if you want to use the default PS, you would declare a template like this:
```yaml
- replicas: 2
      tfReplicaType: PS
      tfImage: tensorflow/tensorflow:1.3.0
```
So it's  already easily customizable.
I chose not to set any default for `tfImage` because it case of version mismatch between the PS and the other nodes it might be quite hard for the users to understand where the issue comes from.
Let me know if you would rather have a default.

Reply:
So tfImage is specified at the Replica level? Why not make it a property of the job instead? This way we only have to specify it once and we can reuse it for the TensorBoard replica and the PS. If you want to specify a custom image at the Replica level you can already do that by filling out a PodTemplateSpec although it would be more verbose.